contracts/scripts/staking/airDrop.js

```diff
@@ -125,8 +126,10 @@ async function main() {
     console.log(`Usage: node airDrop.js <inputJSONFile> <outputJSONFile>`);
   }
 
+  const contractAddress = (await ethers.getContract("OGNStakingProxy")).address;
```

Comment:
Should this actually be OGNStaking rather than the proxy?
Depends on what address(this) returns in Solidity (see line 319 in SingleAssetStaking.sol)